packages/angular/src/schematics/init/init.ts

```diff
@@ -1,29 +1,83 @@
 import {
-  chain,
-  externalSchematic,
-  noop,
-  Rule,
-  schematic,
+  addDependenciesToPackageJson,
+  readJson,
+  readWorkspaceConfiguration,
   Tree,
-} from '@angular-devkit/schematics';
-import {
-  addDepsToPackageJson,
+  updateWorkspaceConfiguration,
+  writeJson,
   formatFiles,
-  readJsonInTree,
-  setDefaultCollection,
-  updateJsonInTree,
-  updateWorkspace,
-} from '@nrwl/workspace';
+} from '@nrwl/devkit';
+import { jestInitGenerator } from '@nrwl/jest';
+import { wrapAngularDevkitSchematic } from '@nrwl/devkit/ngcli-adapter';
+import { cypressInitGenerator } from '@nrwl/cypress';
+
+import { Schema } from './schema';
 import {
   angularVersion,
   jestPresetAngularVersion,
   rxjsVersion,
 } from '../../utils/versions';
-import { Schema } from './schema';
 import { E2eTestRunner, UnitTestRunner } from '../../utils/test-runners';
 
-const updateDependencies = (): Rule =>
-  addDepsToPackageJson(
+export default async function (host: Tree, options: Schema) {
+  setDefaults(host, options);
+  addPostInstall(host);
+  updateDependencies(host);
+
+  await addUnitTestRunner(host, options);
+
+  addE2ETestRunner(host, options);
+
+  if (!options.skipFormat) {
+    await formatFiles(host);
+  }
+}
+
+function setDefaults(host: Tree, options: Schema) {
+  const workspace = readWorkspaceConfiguration(host);
+
+  workspace.generators = workspace.generators || {};
+  workspace.generators['@nrwl/angular:application'] = {
+    style: options.style,
+    linter: options.linter,
+    unitTestRunner: options.unitTestRunner,
+    e2eTestRunner: options.e2eTestRunner,
+    ...(workspace.generators['@nrwl/angular:application'] || {}),
+  };
+  workspace.generators['@nrwl/angular:library'] = {
+    style: options.style,
+    linter: options.linter,
+    unitTestRunner: options.unitTestRunner,
+    ...(workspace.generators['@nrwl/angular:library'] || {}),
+  };
+  workspace.generators['@nrwl/angular:component'] = {
+    style: options.style,
+    ...(workspace.generators['@nrwl/angular:component'] || {}),
+  };
+
+  workspace.cli = workspace.cli
+    ? { ...workspace.cli, defaultCollection: '@nrwl/angular' }
+    : { defaultCollection: '@nrwl/angular' };
+  updateWorkspaceConfiguration(host, workspace);
+}
+
+function addPostInstall(host: Tree) {
+  const pkgJson = readJson(host, 'package.json');
```

Comment:
You could use the `updateJson` utility from `@nrwl/devkit`, it's pretty handy for this.